net/golioth/Kconfig

```diff
@@ -41,6 +41,8 @@ config GOLIOTH_FW
 
 config GOLIOTH_RPC
 	bool "Remote Procedure Call"
+	default y
```

Comment:
I like all the fixes and improvements **besides** enabling GOLIOTH_RPC and GOLIOTH_SETTINGS by default. IMO those should be enabled per application need, same as GOLIOTH_SYSTEM_CLIENT and GOLIOTH_FW.

Reply:
That's fine with me. I'll remove the `default y` lines.